lib/index.js

```diff
@@ -1,7 +1,6 @@
-'use strict'
+import {Transform} from 'node:stream'
+import {Buffer} from 'node:buffer'
```

Comment:
Same as above.  webpack is the reason for the explicit Buffer import.

Reply:
removed `node:`